doc/source/using-baselines.rst

```diff
@@ -1,52 +1,54 @@
 Using Baselines
 ===============
 
+.. warning::
+  We are changing the way we structure the Flower baselines. While we complete the transition to the new format, you can still find the existing baselines and use them: `baselines (old) <https://github.com/adap/flower/tree/main/baselines/flwr_baselines>`_.
+  Currently, you can make use of baselines for `FedAvg <https://github.com/adap/flower/tree/main/baselines/flwr_baselines/flwr_baselines/publications/fedavg_mnist>`_, `FedProx <https://github.com/adap/flower/tree/main/baselines/flwr_baselines/flwr_baselines/publications/fedprox_mnist>`_, `FedOpt <https://github.com/adap/flower/tree/main/baselines/flwr_baselines/flwr_baselines/publications/adaptive_federated_optimization>`_, `FedBN <https://github.com/adap/flower/tree/main/baselines/flwr_baselines/flwr_baselines/publications/fedbn/convergence_rate>`_, and `LEAF-FEMNIST <https://github.com/adap/flower/tree/main/baselines/flwr_baselines/flwr_baselines/publications/leaf/femnist>`_. 
+
+  The documentation below has been updated to reflect the new way of using Flower baselines.
 
 Structure
 ---------
 
-All baselines are available in the directory `baselines <https://github.com/adap/flower/blob/main/baselines>`_. This directory has two different files:
+All baselines are available in the directory `baselines <https://github.com/adap/flower/blob/main/baselines>`_ with each baseline directory being fully self-contained in terms of source code. In addition, each baseline uses its very own Python environment as designed by the contributors of such baseline in order to replicate the experiments in the paper. Each baseline directory contains the following structure: 
 
 .. code-block:: shell
 
-    - pyproject.toml
-    - requirements.txt
+    flower/baselines/<baseline-name>/
+                          ├── README.md
+                          ├── pyproject.toml
+                          └── <baseline-name>
```

Comment:
I replaced the `-` with `_` to point users in the right direction (since `-` are not allowed in Python package names):
```suggestion
    flower/baselines/<baseline_name>/
                          ├── README.md
                          ├── pyproject.toml
                          └── <baseline_name>
```